.eslintrc

```diff
@@ -4,9 +4,10 @@
   ],
   "env": {
     "es6": true,
-    "node": true,
+    "node": false,
+    "jest": false,
     "browser": true,
```

Comment:
Can we also enable the browser env only for files in `lib/browser/`?

Reply:
Good call. That's done, the linter does complain about the use of `window`, but nothing a `// eslint-disable-line` comment can't fix.